modules/understanding-vulnerability-scores.adoc

```diff
@@ -14,8 +14,14 @@ For example, `6.5 (v3)`.
 +
 [NOTE]
 ====
-CVSS v3 scores are only available if you are using the StackRox Scanner version 1.3.5 and later or Scanner V4.
+CVSS v3 scores are only available if you are using the StackRox Scanner version 1.3.5 and later or Scanner V4 (Technology Preview).
 ====
++
+--
+:FeatureName: Scanner V4
```

Comment:
I know I said I wouldn't point out any more, but I feel like as many of the TP warnings you've added are overkill. Like here and other places when you're just mentioning it in passing, that seems like it's probably not necessary.

Seems like it could be kept to more of the high impact places where they are making the decisions to actually use it or not.